react/src/component/Search.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Form } from 'react-bootstrap';
+import { useDebouncedCallback } from 'use-debounce';
+
+export default function Search({searchMovieByTitle}) {
+
+    
+
+    const onChange = (title) => {
+      searchMovieByTitle(title);
+    }
+
+    const debounced = useDebouncedCallback(onChange, 750);
```

Comment:
```suggestion
    const debounced = useDebouncedCallback(searchMovieByTitle, 750);
```

Moze direkto ovako onchange je sada visak

Reply:
Probao sam to i juce, ali puca, mora izgleda ovako